examples/0ctf_trace/solve.py

```diff
@@ -53,7 +53,9 @@ def main():
             'base_addr': 0x400770,
             'arch': 'mipsel',
         },
-    })
+    },
+    auto_load_libs=False,
+    )
```

Comment:
The formatting on this constructor is not great, making adding new arguments not great. I think indenting like this would be better:
```py
    project = angr.Project(
        "./data.bin",
        load_options={
            "main_opts": {
                "backend": "blob",
                "base_addr": 0x400770,
                "arch": "mipsel",
            },
        },
        auto_load_libs=False,
    )
```